account_invoice_batches/__manifest__.py

```diff
@@ -14,6 +14,7 @@
         "l10n_es_facturae",
         "account_invoice_report_service",
         "sale_order_invoicing_grouping_criteria",
+        "account_move_service",
```

Comment:
We cannot make this dependent, we have to allow having batches without move service